clients/cobol-lsp-vscode-extension/src/services/copybook/CopybookDownloadService.ts

```diff
@@ -324,12 +308,10 @@ export class CopybookDownloadService implements vscode.Disposable {
     errors: Set<string>,
     cp: CopybookProfile,
   ) {
-    errors.delete(
-      CopybookDownloadService.getFilenameWithoutExtension(cp.getCopybook()),
-    );
+    errors.delete(this.getFilenameWithoutExtension(cp.getCopybook()));
     this.completedDownload++;
     const downloadPercent = Math.round(
-      (this.completedDownload / this.totalDownload) * 100,
+      (this.completedDownload / CopybookDownloadService.totalDownload) * 100,
```

Comment:
`completedDownload` is static as well, unify the `this`, `CopybookDownloadService` usage.

Reply:
should resolve : 388196d89add26b847152c098f9cc879949bc1ce